libnavui-app/src/main/java/com/mapbox/navigation/ui/app/internal/SharedApp.kt

```diff
@@ -40,24 +43,30 @@ object SharedApp {
     val routeStateController = RouteStateController(store)
     val routePreviewStateController = RoutePreviewStateController(store)
     private val navigationObservers: Array<MapboxNavigationObserver> = arrayOf(
-        destinationStateController,
-        tripSessionStarterStateController,
-        audioGuidanceStateController,
-        locationStateController,
         routeStateController,
-        routePreviewStateController,
         cameraStateController,
+        locationStateController,
         navigationStateController,
+        destinationStateController,
+        routePreviewStateController,
+        audioGuidanceStateController,
+        tripSessionStarterStateController,
     )
 
     fun setup(
```

Comment:
Since this is used by Android Auto, which is a separate library, let's add `@JvmOverloads` to prevent binary incompatibilities. 
```suggestion
    @JvmOverloads
    fun setup(
```